python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -270,6 +271,50 @@ def build(
         )
         return feature_view
 
+    def get_step_size(self) -> timedelta:
+        """
+        Returns the step size of the sliding window features in the feature view. All
+        the sliding window features must have the same step size.
+        """
+        step_size_set = set(
+            [
+                feature.transform.step_size
+                for feature in self.get_resolved_features()
+                if isinstance(feature.transform, SlidingWindowTransform)
+            ]
+        )
+        if len(step_size_set) == 0:
+            raise FeathubException("There must be at least one sliding window feature.")
```

Comment:
It might be better to move the checks to the constructor of SlidingFeatureView, so users can get the exception immediately after the class is created.